arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1660 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-02-03",
+    "title": "iotHubClient",
+    "description": "API for using the IoT Hub Features."
+  },
+  "host": "management.azure.com",
+  "schemes": ["https"],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}": {
+      "get": {
+        "tags": ["GET"],
+        "summary": "Get an IotHub.",
+        "description": "Get an IotHub.",
+        "operationId": "IotHubResource_Get",
+        "consumes": [],
+        "produces": ["application/json", "text/json"],
+        "parameters": [{
+          "name": "resourceName",
+          "in": "path",
+          "description": "The name of the resource.",
+          "pattern":  "^(?![0-9]+$)(?!-)[a-zA-Z0-9-]{2,49}[a-zA-Z0-9]$",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/subscriptionId"
+        }, {
+          "name": "resourceGroupName",
+          "description": "The name of the resource group.",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/api-version"
+        }],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/IotHubDescription"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "put": {
+        "tags": ["PUT"],
+        "summary": "Create or update an IotHub.",
+        "description": "Create or update an IotHub.",
+        "operationId": "IotHubResource_CreateOrUpdate",
+        "consumes": ["application/json", "text/json"],
+        "produces": ["application/json", "text/json"],
+        "parameters": [{
+          "$ref": "#/parameters/subscriptionId"
+        }, {
+          "name": "resourceGroupName",
+          "in": "path",
+          "description": "The name of the resource group.",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "resourceName",
+          "in": "path",
+          "description": "The name of the resource.",
+          "required": true,
```

Comment:
The main place where it made sense is in the schema definition. Added it there and removed from here.     "Resource": { 
      "properties": { 
        "id": { 
          "readOnly": true, 
          "type": "string", 
          "description": "The Resource Id." 
        }, 
        "name": { 
          "readOnly": true, 
          "type": "string",